CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 This project adheres to [Semantic Versioning Scheme](http://semver.org)
 
+## [v0.11.0] 2018-11-26
```

Comment:
I DELIVER 0.11.0 TO MY PEOPLE........FROM THE FUTURE!

Reply:
LOL